gov2/secretsmanager/README.md

```diff
@@ -0,0 +1,70 @@
+# AWS SDK for Go V2 code examples for AWS Secrets Manager 
+
+## Purpose
+
+These examples demonstrate how to perform several AWS Secrets Manager
+operations using version 2 of the AWS SDK for Go.
+
+AWS Secrets Manager helps you to securely encrypt, store, and retrieve credentials for your databases
+and other services. Instead of hardcoding credentials in your apps, you can make calls to Secrets Manager
+to retrieve your credentials whenever needed. Secrets Manager helps you protect access to your IT resources
+and data by enabling you to rotate and manage access to your secrets. 
```

Comment:
Shorter blurb preferred--I'd keep just the first sentence. Also, make it italic to set it off a bit.